Tests/Testo/Invoice/InvItem/InvItemServiceTest.php

```
@@ -0,0 +1,1175 @@
+        Assert::same(1.0, $model->getDiscountAmount());
+        Assert::same(1, $model->getOrder());
+        Assert::same(5, $model->getSoItemId());
+        Assert::same(100, $model->reqInvId());
+        Assert::same('X1', $model->getPeppolPoItemid());
+        Assert::same('L1', $model->getPeppolPoLineid());
+        Assert::same('1', $model->getBelongsToVatInvoice());
+        Assert::same($taxRateCtor, $model->getTaxRate());
+        Assert::same($productCtor, $model->getProduct());
+        Assert::same($taskCtor, $model->getTask());
+        Assert::same($invEntity, $model->getInv());
+    }
```

PHPMD (2.15.0)

[warning] The method addInvItemProductHappyPathAppliesAllRelationsAndCreatesAmount() has 148 lines of code. Current threshold is set to 100. Avoid really long methods. (undefined)

(ExcessiveMethodLength)